File: chess.c
#include <stdio.h>


void print_board(int row_no,int col_no)
{
    if(row_no == 1 && col_no==1 || row_no == 8 && col_no==1) {
        printf("♖");
        printf(".");
    } else if(row_no == 1 && col_no==2 || row_no == 8 && col_no==2) {
        printf("♘");
        printf(".");
    } else if(row_no == 1 && col_no==3 || row_no == 8 && col_no==3) {

        printf("♗");
        printf(".");
    } else if(row_no == 1 && col_no==4 || row_no == 8 && col_no==4) {
        printf("♕");
        printf(".");
    } else if(row_no == 1 && col_no==5 || row_no == 8 && col_no==5) {
        printf("♔");
        printf(".");
    } else if(row_no == 1 && col_no==6 || row_no == 8 && col_no==6) {
        printf("♗");
        printf(".");
    } else if(row_no == 1 && col_no==7|| row_no == 8 && col_no==7) {
        printf("♘");
        printf(".");
    } else if(row_no == 1 && col_no==8 || row_no == 8 && col_no==8) {
        printf("♖");
        printf(".");
    } else if(row_no==3 || row_no==4||row_no==5||row_no==6) {
        printf(" . ");
    }
}

void main()
{
    int row,col,row_input,col_input;
    for(row=1; row<9; row++) {
        for(col=1; col<9; col++) {
            print_board(row,col);
// all important piecss printed above
            if(row ==2 || row==7) {
                printf("♙");
                printf(".");

            }

        }
        printf("\n");
    }
    printf("{Player-1}Enter the position of piece to move\n");
    printf("{Player-1}Enter the row_no:");
    scanf("%d",&row_input);
    printf("{Player-1}Enter the col_no:");
    scanf("%d",&col_input);


}

File: chess.py

board = [
    [" ","1"," 2"," 3"," 4"," 5"," 6"," 7"],
    ["1","♜","♞","♝","♚","♛","♝","♞","♜"],
    ["2","♟","♟","♟","♟","♟","♟","♟","♟"],
    ["3","."," ."," ."," ."," ."," ."," ."," .","."],
    ["4","."," ."," ."," ."," ."," ."," ."," .","."],
    ["5","."," ."," ."," ."," ."," ."," ."," .","."],
    ["6","."," ."," ."," ."," ."," ."," ."," .","."],
    ["7","♙","♙","♙","♙","♙","♙","♙","♙"],
    ["8","♖","♘","♗","♕","♔","♗","♘","♖"]
         ]

def print_board(get_pos1,get_pos2,get_pos3,get_pos4):
    piece = board[get_pos1][get_pos2]
    board[get_pos3][get_pos4] = piece
    board[get_pos1][get_pos2] = "."
    
    for row in range(0,len(board)-1):
        for col in range(0,len(board)-1):
            if row == 0 or row == 8 and row == 2 or row == 7:
                print(board[row][col],end = "  ")
            else:
                print(board[row][col],end = "  ")
        print("\n")
        

def get_input():
    #for i in range(0,len(board)):
       # print(board[i])
    for row in range(0,len(board)-1):
        for col in range(0,len(board)-1):
            if row == 0 or row == 8 and row == 2 or row == 7:
                print(board[row][col],end = "  ")
            else:
                print(board[row][col],end = "  ")
        print("\n")
    print("{player-1}")
    pos1 = int(input("enter row no.:"))
    pos2 = int(input("enter col no.:"))
    pos3 = int(input("where to move enter row no.:"))
    pos4 = int(input("where to move enter col no.:"))
    print("\n")
    print_board(pos1,pos2,pos3,pos4)
    print("{player-2}")
    pos1 = int(input("enter row no.:"))
    pos2 = int(input("enter col no.:"))
    pos3 = int(input("where to move enter row no.:"))
    pos4 = int(input("where to move enter col no.:"))
    print("\n")
    print_board(pos1,pos2,pos3,pos4)
    print("\n")

choice = "exit"
a = input("enter choice:")
while(a != choice):
    get_input()
